ongebo/java-coders-nairobi
# #158792524 Create mockup designs for the project

### What does this PR do?
Creates mockup designs for the project
#### Description of the task to be completed?
- Create a `wireframes` in the root directory
- Add .png images for the screen designs
- Add a .gif file to illustrate user interactions
#### How should this be manually tested?
- On this branch, open the `wireframes` directory
- Open the .png files to see the UI design for the app
- Open the .gif file to see how the user would interact with the app
#### Screenshots
![image](https://user-images.githubusercontent.com/34346070/53497531-6fc23180-3ab5-11e9-9ed8-5e68f35124b8.png)

![image](https://user-images.githubusercontent.com/34346070/53497523-69cc5080-3ab5-11e9-9375-fac00b3cb7c8.png)

![image](https://user-images.githubusercontent.com/34346070/53497499-58834400-3ab5-11e9-9548-6ca65331f913.png)

#### What are the relevant Pivotal Tracker stories?
[#158792524](https://www.pivotaltracker.com/story/show/158792524)
